structure/0048-communications-team.md

```diff
@@ -0,0 +1,71 @@
+# Communications Team
+
+This document has been made according to the process outlined in [RFC
+0006][RFC6].
+
+## Summary
+
+The Communications Team is the team in charge of communications within and
+outside the Quilt Project. This team is also responsible for working with the
+development teams to communicate the current status of development progress
+with the community and with any partners. In essence, it may be thought of as
+a "public relations department" of sorts.
+
+## Motivation
+
+This team is needed because developers would rather focus on development than
+on public relations, status updates, or team to-do lists. Historically, this
+has been managed by the Community Team. However, that team's scope is already
+wide enough. Communications fall outside the realm of the Quilt community, and
+thus this should have its own separate team.
+
+## Explanation
+
+The Communications Team will be responsible for all communications outside of
+and within the Quilt Project. This is difficult to define without providing
+specific examples, which include:
+
+- writing and publishing blog posts
+- managing the official QuiltMC social medias, with the exception of Discord,
+  and not including moderation
```

Comment:
Does this mean Quilt moderators will have to be added to every single external media Quilt manages? In the case of, say, Reddit it could be doable although perhaps a bit hard to do, but for blocking Twitter users it isn't really possible. I think for such cases, the communication team should be capable of moderating their own spaces, assuming it is done after polling the moderation team. Another option would be to have the comm team not be allowed to moderate by themselves but be able to prompt and relay actions of the mod team when they can't action it themselves. A dedicated channel could be added on Discord for that purpose.

Reply:
I think communication team should be trusted to moderate their own platforms. communication team's "jurisdiction" is social media like twitter, tumblr, blog posts, etc. social media sites like Reddit or Discord with more of a community focus should be run by the community/moderation team, but could be run temporarily by communication team, with the caveat that they'd hand over control if for example, a quilt subreddit becomes an active discussion area. also, moderation/community team should be able to override the communication team on moderation decisions if necessary.

Might make sense to make communication team a sub-team of moderation/community team. Like how Catholic Cardinals aren't a "higher" rank than Bishops, they're bishops with additional responsibilities and titles.

Reply:
I considered making it a subteam of the community team, but then decided that it's really not in scope for that team. I mentioned the reasons for this in the document; community team is mostly moderation and events, and it shouldn't really be the responsibility of moderation and event staff to do PR. Stuff like the content of the website just... isn't a community thing.
Anyway, I do agree that communications team should have the ability to moderate when necessary, but it shouldn't be their primary responsibility. I'll work on adding that when I can.